### DISCUSSION TITLE
Release Notes - 4.0.0 Beta

### DISCUSSION THREAD

**Discussion by dahall** [dahall/Vanara#435] [General]
* Nullable notation added to every project
* Lots of **breaking changes** from that nullability as I adjusted code based on new constraints
* Use of global `using` statements
* Addition of .NET 7.0
* Updates and additions to unit tests

This will stay in beta status for about two months while I collect feedback and fix problems. I hope you'll try it and give me a hand finding the errors and inconsistencies.

<hr /><em>This discussion was created from the release <a href='https://github.com/dahall/Vanara/releases/tag/v4.0.0-beta'>Release Notes - 4.0.0 Beta</a>.</em>

**Comment by tajbender** [comment]
Nice, sounds great.

I'll give it a try the next days 🥇

**Reply by dahall** [reply]
Thanks. Your review would mean a lot given your use.

**Reply by tajbender** [reply]
Yeah, I had to have a break the last months... or years.

I suffer from maniac depression, so I'm on and off in the business...

But currently, I've joined the new Hackathon using .net 7, so this matches very well 😁 

Thanks, David.
